scripts/plot_run_duration_over_time.json

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+# Takes a single benchmark output JSON and plots the duration of item executions over time
+
+# TODO add to requirements.txt
```

Comment:
What is the best way to do that? I see that requirements.txt also holds the version numbers. Do I just take the current one?

Reply:
`pip freeze`